app/scripts/sources.json

```diff
@@ -0,0 +1,157 @@
+{
+    "itemHash":{
+       "488703545":{"sourceHashes":[1662396737]},
```

Comment:
I'm not super comfortable with the idea of it being our responsibility to  augment or correct Bungie's data.

That said, this whole file could be much simpler, right?

```json
{
  "488703543": [1662396737],
  "4063984059": [1662396737],
}
```